deepmd/tf/model/model.py

```diff
@@ -693,6 +693,7 @@ def __init__(
                 spin=self.spin,
                 ntypes=self.descrpt.get_ntypes(),
                 dim_descrpt=self.descrpt.get_dim_out(),
+                embedding_width=self.descrpt.get_dim_rot_mat_1(),
```

Comment:
There is an issue here. `get_dim_rot_mat_1` is unavailable for `loc_frame`. The current behavior is to throw `NotImplementEderror`. The energy fitting doesn't need this argument, while `loc_frame` only supports the energy fitting.

I am not sure what the best design is here...

Reply:
how about we add an check here based on the fitting_net type?

Reply:
Btw, I think `se_e2_r` doesn't support it in theory.